converter/__init__.py

```diff
@@ -1,6 +1,10 @@
-from .converter import celsius_to_fahrenheit, fahrenheit_to_celsius
+from .converter import celsius_to_fahrenheit, fahrenheit_to_celsius, ConverterContext
 from .exception import TemperatureError, ConverterError
 from .utils import is_float
 
 
-__all__ = ["celsius_to_fahrenheit", "fahrenheit_to_celsius", "is_float", "TemperatureError", "ConverterError"]
+__all__ = [
+    "celsius_to_fahrenheit", "fahrenheit_to_celsius", "is_float",
+    "TemperatureError", "ConverterError",
```

Comment:
@MariaMaximova , please add `ValueError` here, as it might be usefull to export it too.

Reply:
`ValueError` was [added](https://github.com/MariaMaximova/convert_temp/pull/19/commits/783a509aa1c26b0826ad164b19370b38f1a70377#diff-a8bdc5cce37ceb169dc32db55c9b3aaa210c0d831a2207db82cc78749b6c2bc0)